src/realmd/CMakeLists.txt

```diff
@@ -77,7 +77,7 @@ if(UNIX)
 endif()
 
 if(APPLE)
-  set(EXECUTABLE_LINK_FLAGS "-framework Carbon ${EXECUTABLE_LINK_FLAGS}")
+  set(EXECUTABLE_LINK_FLAGS "-Wl,-no_fixup_chains -framework Carbon ${EXECUTABLE_LINK_FLAGS}")
```

Comment:
I actually fixed that problem a while ago (see cmangos/mangos-tbc@111e48d17713e4f0dd9b1ac02d5e4f43b3fdd993 and the merge of that to other cores this week), so we don't need to sweep that issue under the carpet any more :wink: